spec/index.html

```diff
@@ -167,6 +169,69 @@ <h3>Resources and Statements</h3>
       exists, without explicitly naming it.</p>
   </section>
 
+  <section id="section-quoted-triples">
+    <h3>Quoted Triples</h3>
+
+    <p>Sometimes, we want the <a>subject</a> or <a>object</a> of a <a>triple</a> to refer to another triple.
+      For example, the statement "according to `employee22`, `employee38` has a `jobTitle` of 'Assistant Designer'"
+      can be modeled as a <a>triple</a> with "according to" as its <a>predicate</a>,
+      `employee22` as its <a>object</a>,
+      and another triple as its <a>subject</a>,
+      namely the triple "`employee38` has a `jobTitle` of 'Assistant Designer'".
+      This use of a triple as the subject or object resource of another triple,
+      so that we can say things about the triple so used, is known as
+      <a data-cite='RDF12-SEMANTICS#dfn-reification'>reification</a>.
+      The concept of reification has always been part of RDF,
+      but expressing it in most RDF concrete syntaxes such as Turtle and N-Triples,
+      as well as processing or querying it with SPARQL,
+      has been verbose and cumbersome [[RDF-STAR-FOUNDATION]].</p>
+
+    <p>This specification extends RDF's conceptual data model,
+      which provides a compact alternative to <a data-cite='RDF12-SEMANTICS#dfn-reification'>standard RDF reification</a>.
+      This model enables the creation of concise <a>triples</a> that reference other triples
+      as <a>subject</a> and <a>object</a> <a>resources</a>.
+      Triples that include a <a>triple</a> as a <a>subject</a> or an <a>object</a> are known as
+      <a>complex triples</a>.
+      The following graph shows the example <a>complex triples</a> in the Turtle syntax,
+      which uses double angle brackets to enclose a <a>triple</a> serving as a <a>subject</a> or <a>object</a> resource:</p>
+
+    <pre id="ex-quoted-triple" class="example"
+         data-transform="updateExample"
+         data-content-type="text/turtle">
+      <!--
+      PREFIX :    <http://www.example.org/>
+
+      :employee38                                      :familyName "Smith" .
+      << :employee38 :jobTitle "Assistant Designer" >> :accordingTo :employee22 .
+      -->
```

Comment:
If retaining Turtle, with or without diagram, a few other tweaks to help new people comprehend the example —
* it's easy to overlook or misunderstand the bare `:`; hence, `ex:`
* removing the extra line indents within the `<pre>`
* putting two space chars (instead of one) between columns
```suggestion
PREFIX  ex:  <http://www.example.org/>

ex:employee38                                           ex:familyName   "Smith" .
<< ex:employee38  ex:jobTitle  "Assistant Designer" >>  ex:accordingTo  ex:employee22 .
      -->
```